roles/disaster_recovery/tasks/clean/remove_valid_filtered_master_domains.yml

```diff
@@ -1,24 +1,54 @@
 ---
 - name: Remove valid storage domain main block
   block:
-    - name: Fetch active/maintenance/detached storage domain for remove
+    - name: Fetch active storage domain for remove
+      ovirt_storage_domain_info:
+        pattern: >
+           name={{ storage['dr_' + dr_source_map + '_name'] }} and
+            datacenter={{ storage['dr_' + dr_source_map + '_dc_name'] }} and {{ dr_active_domain_search }}
+        auth: "{{ ovirt_auth }}"
+      register: storage_domain_info_active
+
+    - name: Fetch maintenance storage domain for remove
+      ovirt_storage_domain_info:
+        pattern: >
+          name={{ storage['dr_' + dr_source_map + '_name'] }} and
+            datacenter={{ storage['dr_' + dr_source_map + '_dc_name'] }} and {{ dr_maintenance_domain_search }}
+        auth: "{{ ovirt_auth }}"
+      register: storage_domain_info_maintenance
+
+    - name: Fetch detached storage domain for remove
```

Comment:
This section (lines 20-26) seems duplicate (same as lines 4-10) + wrong name?
AFAIU lines 20-26 should be removed.